nbrsessionproxy/handlers.py

```diff
@@ -67,112 +54,50 @@ class RSessionProxyHandler(IPythonHandler):
 
     def initialize(self, state):
         self.state = state
+        self.port = 9797
 
     def rsession_uri(self):
-        return '{}proxy/{}/'.format(self.base_url, self.state['port'])
-
-    def gen_response(self, proc):
-        response = {
-            'pid': proc.pid,
-            'url':self.rsession_uri(),
-        }
-        return response
-        
-    def get_client_id(self):
```

Comment:
Now that nbserverproxy does auth right, I guess get_client_id() won't be necessary! It wasn't being used, but I thought it'd be be necessary for security on multi-user servers.